pkg/storaged/btrfs/subvolume.jsx

```diff
@@ -242,7 +245,63 @@
     });
 }
 
-export function make_btrfs_subvolume_page(parent, volume, subvol) {
+function dirname(path) {
+    const i = path.lastIndexOf("/");
+    if (i < 0)
+        return null;
+    else
+        return path.substr(0, i);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-20067-20240221-114320-9bf547b2-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)